src/Nelmio/Alice/Instances/Processor/ProcessableInterface.php

```diff
@@ -33,4 +33,11 @@ public function valueMatches($regexString);
      * @return string
      */
     public function getMatch($name);
+
+    /**
+     * return all matches
+     *
+     * @return array|string[]
```

Comment:
Well, actually I don't know, if its jut `string[]` yeah, if it can return something else we have to specify it too, but `array` is way too unclear